webcrypto/crypto_test.go

```diff
@@ -10,10 +11,10 @@ import (
 func TestGetRandomValues(t *testing.T) {
 	t.Parallel()
 
-	ts := newTestSetup(t)
+	ts, err := newConfiguredRuntime(t)
+	require.NoError(t, err)
```

Comment:
nit: probably better to just `require.NoError` in the `newConfiguredRuntime` - I don't think there is a case for it returning error and us not stopping a test